python/src/aiconfig/editor/server/server_utils.py

```diff
@@ -56,7 +57,7 @@ class ServerMode(Enum):
 
 class EditServerConfig(core_utils.Record):
     server_port: int = 8080
-    aiconfig_path: Optional[str] = None
+    aiconfig_path: str = "my_aiconfig.aiconfig.json"
```

Comment:
can we just call this `my_aiconfig.json`. Having two separate `.` is confusing to a lot of n00bs

Reply:
I thought the standard was going to be .aiconfig.json? cc @saqadri ?